android/src/main/java/app/notifee/core/model/NotificationAndroidStyleModel.java

```diff
@@ -189,8 +192,9 @@ private Task<NotificationCompat.Style> getBigPictureStyleTask(Executor executor)
             Bitmap largeIconBitmap = null;
 
             try {
-              largeIconBitmap =
-                  Tasks.await(ResourceUtils.getImageBitmapFromUrl(largeIcon), 10, TimeUnit.SECONDS);
+              largeIconBitmap = 
+                  ResourceUtils.getImageBitmapFromUrl(largeIcon)
```

Comment:
why do we not need to await this anymore?

Reply:
as far as I remember it still does await, just changed from "Tasks.await" to ".get"
correct me if I'm wrong though
so in this case check the next line of L196, you will see ` .get(10, TimeUnit.SECONDS);`